### PR TITLE
Added 'expected output' to PR template.

### PR DESCRIPTION
**Pull request by ElijahCaine**
This will help further clarify some our PR work-flow. Thoughts?

@osuosl/devs

### CODE REVIEW

**Comment by matthewrsj** [comment]
If we are separating out Expected Output, we should remove `See Z` on line 29.

**Comment by matthewrsj** [comment]
Or we can just enforce the `See Z` portion of testing, since the expected output is specifically for the test output.

**Comment by ElijahCaine** [comment]
@matthewrsj Pretty much 100% of the PR template is suggestion so keeping both of them is fine. If the PR only necessitates a `see X` then the author can scrap the `Expected output` section.

I know we have some tests that require a bit more explanation and putting the section in there should only make our PR work-flow more clear.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -28,4 +28,15 @@ fixes issue #xx
 2. Do Y.
 3. See Z.
 
+### Epected Output.
```
Comment (alxngyn) [inline .github/PULL_REQUEST_TEMPLATE.md]: Spelling error (Expected)

**Comment by alxngyn** [comment]
+1 since I already do this already, having a template will make testing more clear imo

**Comment by matthewrsj** [comment]
Alright, no problem. @ElijahCaine can you change the last `lsit` misspellings to `list` while you are here (line 11)? +1 when that's done.

**Comment by ElijahCaine** [comment]
Feedback fixed.

**Comment by matthewrsj** [comment]
@ElijahCaine did you mean to add 3 more files?

**Comment by matthewrsj** [comment]
+1

**Pull request merged**